data/sreejit-sengupto.json

```
@@ -1,26 +1,72 @@
 {
-  "name": "SREEJIT SENGUPTA",
-  "bio": "Open Source Enthusiast! On and into Web Development. Aspiring Open Sorcerer. Constantly trying to Upskill myself. Learning everyday!",
+  "name": "Sreejit Sengupta",
+  
+  "bio": "JavaScript | React | Frontend Developer | Tailwind | Blogger at **Hashnode**"
+  },
+{
+  "tags": ["Blogger", "Javascript", "Tailwind", "React", "Tweeter"]
+},
+"socials": [
+  { "icon": "FaTwitter", "url": "https://twitter.com/SreeSen03" },
+  { "icon": "FaGithub", "url": "https://github.com/Sreejit-Sengupto" },
+  { "icon": "Hashnode", "url": "sreesen.hashnode.com" },
+  { "icon": "LinkedIn", "url": "https://linkedin.com/in/sreesen03"}
+],
   "links": [
     {
-      "name": "Catch me on GitHub",
+      "name": "GitHub",
       "url": "https://github.com/sreejit-sengupto",
       "icon": "FaGithub"
     },
     {
-      "name": "Catch me on Twitter",
+      "name": "Twitter",
       "url": "https://twitter.com/SreeSen03",
       "icon": "FaTwitter"
+    },
+    {
+      "name": "LinkedIn",
+      "url": "https://linkedin.com/in/sreesen03",
+      "icon": "FaLinkedin"
+    },
+    {
+      "group": "Blogs",
+      "name": "TechStuff Blogs",
+      "url": "sreesen.hashnode.dev",
+      "icon": "FaHashnode"
     }
   ],
   "milestones": [
     {
-      "title": "No such milestones but I Learnt!",
-      "date": "October 2022",
-      "icon": "FaBook",
+      "title": "Started writing blogs on Hashnode",
+      "date": "October 7, 2022",
+      "icon": "FaHashnode",
+      "color": "grey",
+      "description": "Wrote my first blog describing three ways to include CSS in an HTML file",
+      "url": "sreesen.hashnode.dev"
+    },
+    {
+      "title": "Cloned Battlegrounds Mobile India's website",
+      "date": "June 24, 2023",
+      "icon": "FaGamepad",
+      "color": "grey",
+      "description": "Cloned the entire website of BGMI using React, React-Router and TailwindCSS"
```

```suggestion
      "description": "Cloned the entire website of BGMI using React, React-Router and TailwindCSS",
```
There is a comma missing.